src/utils/helpers.ts

```diff
@@ -36,9 +36,12 @@ export const parseAgency = (agency: string): string =>
  * @returns true if the date is an upcoming date, false otherwise
  */
 export const isValidDate = (d: string) => {
-  const currentDate = new Date();
-  currentDate.setHours(0, 0, 0, 0);
-  if (d !== '' && new Date(d) >= currentDate) {
+  const tomorrow = new Date(); 
```

Comment:
question: do we not want them to be able to select the current day? i feel like if the user has no restrictions on what day they can contact the client, it would feel more intuitive to select the current day as opposed to the next day

Reply:
oh yeah lemme update, I agree. I should probably update the wording for the error feedback too!